frontend/packages/react-app/src/components/atoms/connectors.ts

```diff
@@ -1,5 +1,5 @@
 import { InjectedConnector } from "@web3-react/injected-connector";
 
 export const injected = new InjectedConnector({
-  supportedChainIds: [1, 3, 4, 5, 42],
+  supportedChainIds: [1, 4],
```

Comment:
When the selected chain id is not one of these pushing "connect wallet" button shows nothing. 
IMO we should display something to indicate that the user should switch the network.
And an error occurs connected networks other than Ropsten.

Reply:
OK